mautic/mautic
# Confirm Me - MJML Fixes

a = current major release


| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ X]
#### Description:

This PR adds code to the MJML head, makes the template more stable.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a segment or template email using this template. Edit the text - now you'll see, that font type / size is not changing.
3. Padding is set properly.
